applications/openshift/api-server/api_server_request_timeout/rule.yml

```diff
@@ -40,3 +40,17 @@ ocil: |-
     Run the following command:
     <pre>$ oc get configmap config -n openshift-kube-apiserver -ojson | jq -r '.data["config.yaml"]' | jq '.apiServerArguments["min-request-timeout"]'</pre>
     The output should return <pre>300</pre>.
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
+    entity_check: "at least one"
+    filepath: /api/v1/namespaces/openshift-kube-apiserver/configmaps/config
+    yamlpath: '.data["config.yaml"]'
+    xccdf_variable: var_api_min_request_timeout
+    embedded_data: "true"
+    values:
+    - value: '"apiServerArguments":{.*"min-request-timeout":\["(\d*)"\]'
+      operation: "pattern match"
+      type: "string"
```

Comment:
Let's add an e2e test to ensure this is passing [1]. No remediation should be needed for this one with this default. 

All you need for the test is:

```
---
default_result: PASS
```

[1] https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#writing-e2e-tests-for-specific-rules

Reply:
I will add the test.

Reply:
Done